src/roller/dice.ts

```diff
@@ -568,15 +617,43 @@ export class StackRoller extends GenericRoller<number> {
             }
         }
 
-        const result = [`${rounded}`];
-        if (this.plugin.data.displayResultsInline) {
-            result.unshift(this.inlineText);
+        let result;
+        if (this.expectedValue === ExpectedValue.None && !this.shouldRender) {
+            if (this.showDice) {
+                result = [""];
+            }
+            else {
+                result = ['\xa0'];
+            }
+            if (this.showFormula) {
+                result.unshift(this.original+" -> ");
+            }
+        }
+        else {
+            result = [`${rounded}`];
+            if (this.showFormula) {
+                result.unshift(this.inlineText);
+            }
+        }
+
+        this.expectedValue = ExpectedValue.Roll;
+
+        if (this.displayFixedText) {
+            this.resultEl.setText(this.fixedText);
+        }
+        else {
+            this.resultEl.setText(result.join("") + this.stunted);
         }
-        this.resultEl.setText(result.join("") + this.stunted);
     }
     async onClick(evt: MouseEvent) {
         evt.stopPropagation();
         evt.stopImmediatePropagation();
+        if (evt.altKey) {
```

Comment:
Please use the `getModifierState` api shown [here](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/getModifierState)